### PR TITLE
Fix unbinding of onComplete callback

### PR DESCRIPTION
**Pull request by hjheath**
The `unbind` was broken because `events.off` was not being passed the exact same function as `events.on`

### CODE REVIEW

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-233-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-233-pr-onfido-sdk-ui-onfido.surge.sh

File: src/index.js [hunk 1]
```diff
@@ -30,24 +30,17 @@ const Container = ({ options }) =>
...
-
-const unbindCompleteEvent = (options) => {
-  events.off('complete', complete(options))
-}
-
-const rebindCompleteEvent = (newOptions, bindedComplete) => {
-  unbindCompleteEvent(bindedComplete)
-  bindCompleteEvent(newOptions)
+const rebindOnComplete = (newOptions) => {
+  events.removeAllListeners('complete')
```
Comment (stefaniacardenas) [inline src/index.js]: 👍

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-233-pr-onfido-sdk-ui-onfido.surge.sh

File: src/index.js [hunk 2]
```diff
@@ -30,24 +30,16 @@ const Container = ({ options }) =>
...
-  return () => {
-    Tracker.sendEvent('completed flow')
-    options.onComplete()
-  }
-}
-
-const bindCompleteEvent = (options) => {
-  events.on('complete', complete(options))
-}
+const trackOnComplete = () => Tracker.sendEvent('completed flow')
```
Comment (rfreitas) [inline src/index.js]: nice!

File: src/index.js [hunk 3]
```diff
@@ -30,24 +30,16 @@ const Container = ({ options }) =>
...
-const unbindCompleteEvent = (options) => {
-  events.off('complete', complete(options))
+const bindOnComplete = ({onComplete}) => {
+  events.on('complete', onComplete)
 }
 
-const rebindCompleteEvent = (newOptions, bindedComplete) => {
-  unbindCompleteEvent(bindedComplete)
-  bindCompleteEvent(newOptions)
+const rebindOnComplete = (oldOptions, newOptions) => {
```
Comment (rfreitas) [inline src/index.js]: very minor: but you can do deconstructing on both options here.
`({onComplete: oldOnComplete, onComplete})`

**Pull request merged**